demos/hide/hide.go

```diff
@@ -26,8 +24,8 @@ func (game *GameWorld) Setup(w *ecs.World) {
 	render := engi.NewRenderComponent(texture, engi.Point{8, 8}, "guy")
 	collision := &engi.CollisionComponent{Solid: true, Main: true}
 
-	width := texture.Width() * render.Scale.X
-	height := texture.Height() * render.Scale.Y
+	width := texture.Width() * render.Scale().X
```

Comment:
why is it a method now and before was a public property access? To me it makes more sense that components in ECS have simply public access to variables.